assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -71,7 +71,7 @@ const UploadingIndicator = styled.div`
   background: linear-gradient(270deg, #4285f4 0%, #0f0bc8 57%, #4285f4 110%);
   background-size: 400% 400%;
   position: absolute;
-  bottom: 10px;
+  bottom: 4px;
```

Comment:
Looking at #575, the uploading indicator is supposed to be at the very bottom.

```suggestion
  bottom: 0;
```

Reply:
As noted here https://github.com/google/web-stories-wp/issues/7693#issuecomment-847016847

Making it bottom 0, looks strange, as the progress bar overlays the bottom of the image / video. As the bottom of the image / video has rounded corners, the straight progress bar looks odd.

Reply:
<img width="174" alt="Screenshot 2021-05-25 at 11 21 13" src="https://user-images.githubusercontent.com/237508/119482209-823a7c80-bd4b-11eb-9b9f-dfc32f9ff8d5.png">

An example.

Reply:
Can the bottom corners of the progress bar be made round as well then?

Reply:
Indeed it can https://github.com/google/web-stories-wp/pull/7699/commits/cae00da6e27f1bdf814b8b8e64a73213e5f8ee3c. 

<img width="212" alt="Screenshot 2021-05-25 at 15 36 49" src="https://user-images.githubusercontent.com/237508/119517036-297cdb00-bd6f-11eb-81d3-097b6c77226a.png">